src/ui/components/questions/questionsubmissioncomponent.js

```diff
@@ -323,10 +332,12 @@ export default class QuestionSubmissionComponent extends Component {
         questionDescription: formData.questionDescription
       })
         .catch(error => {
+          let errorMessage = this._config.networkErrorText;
```

Comment:
do you think it makes sense to rename networkErrorText to defaultErrorText?

Reply:
that would work too